packages/horizontal-slider-react/package.json

```diff
@@ -32,7 +32,7 @@
     "@babel/runtime": "^7.0.0"
   },
   "peerDependencies": {
-    "react": ">= 16.8.0 < 18",
-    "react-dom": ">= 16.8.0 < 18"
+    "react": ">= 16.8.0 < 19",
```

Comment:
not 19 as well?

Reply:
Was not sure with 19 still being in RC at the moment. Wdyt?

Reply:
I think if you try it out with the RC and it works (I assume it does), personally I'm happy putting <20

Reply:
Should be good now. I kept React 18 for the demo, as switching to RC requires to override the type dependencies which makes it less readable for users.